### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by JaredBeale**
Conformed README.md to the standard-readme, updated package.json to include description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
-4. In the same terminal, run ```npm start``` to get the server running.
-5. Then open another terminal window and go to the client folder by doing ```cd client``` and then run ```npm install``` 
-6. Then run ```npm start``` and go to http://localhost:3000 to open the website.
+# CS24-Crowdsourced-Video-Classification
+Web application to crowdsource the effort to associate emotion labels with video clips.
+
+Xandr, an AT&T company, wants to build a machine learning algorithm to identify emotion in video content. The first step is the collection of a large set of labeled data. This application is designed to facilitate gathering such information by allowing users to view and label video clips.
+
+## Install
+This module requires use of a PostgreSQL database. Create a local one or use an external one.
```
Comment (developerJuiceLozzoc) [inline README.md]: probably should move that elsewhere, seeing how that confuses debuggers, when in fact they need the db url instead.

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "cs24-capstone-crowdsourced-video-classification",
   "version": "1.0.0",
   "author": "Jared Beale, Sam Young, Conner Maddalozzo",
-  "description": "",
+  "description": "Web application to crowdsource the effort to associate emotion labels with video clips.",
   "main": "index.js",
   "license": "ISC",
```
Comment (developerJuiceLozzoc) [inline package.json:7]: we should probably change that to a private license found here https://choosealicense.com/licenses/ the ISC is a general license that permites any use of the software AS IS

**Review by developerJuiceLozzoc — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (JaredBeale) [reply package.json:7]: Switched

File: README.md (hunk 1 above)
Comment (JaredBeale) [reply README.md]: Clarified

**Pull request merged**